applications/nrf_desktop/doc/power_manager.rst

```diff
@@ -1,7 +1,132 @@
+|                                  |                        |                                  |                      | :ref:`board`                       |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_main`          | ``module_state_event`` |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`ble_state`                 | ``ble_peer_event``     |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`usb_state`                 | ``usb_state_event``    |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`hid_state`                 | ``hid_report_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+| :ref:`nrf_desktop_power_manager` | ``power_down_event``   |                                  |                      |                                    |
++----------------------------------+------------------------+                                  +----------------------+------------------------------------+
+|                                  | ``wake_up_event``      |                                  |                      |                                    |
++----------------------------------+------------------------+----------------------------------+----------------------+------------------------------------+
+
+
+External dependencies
+*********************
+
+This module utilizes Zephyr :ref:`zephyr:power_management`.
+It depends on :option:`HAS_SYS_POWER_STATE_DEEP_SLEEP_1` being enabled and selects :option:`DEVICE_POWER_MANAGEMENT` and :option:`SYS_POWER_DEEP_SLEEP_STATES`.
+
+
+Configuration
+*************
+
+The module is enabled by the ``CONFIG_DESKTOP_POWER_MANAGER_ENABLE`` option.
+
+With ``CONFIG_DESKTOP_POWER_MANAGER_TIMEOUT`` configuration option you can set time after which the application will go to low power mode.
+The ``CONFIG_DESKTOP_POWER_MANAGER_STAY_ON`` let system stay idle also when there are no active connections.
+The ``CONFIG_DESKTOP_POWER_MANAGER_ERROR_TIMEOUT`` set the time after which the device is turned off upon an internal error.
```

Comment:
Are these three lines "suboptions" of the enable option? If so, I would introduce them with a sentence ("Once the module is enabled, you can enable one or more of the following options:") and put these three into bullet points.